frontend/language/src/nb.json

```
@@ -642,7 +642,7 @@
   "process_editor.configuration_panel_header_help_text_confirmation": "Du bruker oppgaven Bekreftelse (confirmation) til å vise sluttbrukerne en oppsummering som de kan bekrefte.",
   "process_editor.configuration_panel_header_help_text_data": "Du bruker oppgaven Utfylling (data) hvis du skal samle inn eller vise data.",
   "process_editor.configuration_panel_header_help_text_feedback": "Du bruker oppgaven Tilbakemelding (feedback) når du vil at sluttbrukerne skal vente på en tilbakemelding fra tjenesteeier før de kan gå videre i prosessen eller avslutte den.",
-  "process_editor.configuration_panel_header_help_text_payment": "",
+  "process_editor.configuration_panel_header_help_text_payment": "Du bruker oppgaven Betaling (payment) når du vil at sluttbrukerne skal foreta en betaling.",
```

```suggestion
  "process_editor.configuration_panel_header_help_text_payment": Du bruker oppgaven Betaling (payment) når du vil at sluttbrukerne skal betale for noe.
```
Vurdere å endre til stor forbokstav på stegnavnene i alle stegene (Payment, Feedback, osv)